FirebaseInAppMessaging/Swift/Tests/Integration/FIAMSwiftUI/Shared/ContentView.swift

```diff
@@ -0,0 +1,45 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import FirebaseInAppMessaging
+
+struct ContentView: View {
+  @State var analyticsEvent = ""
```

Comment:
Nit: `@State` variables should also be marked with `private` so they aren't modified outside of this scope.

Reply:
Still open

Reply:
Fixed in 5d8b009.